php/class-customize-snapshot-manager.php

```diff
@@ -202,6 +202,7 @@ public function enqueue_controls_scripts() {
 		// Script data array.
 		$exports = apply_filters( 'customize_snapshots_export_data', array(
 			'inspectLink' => isset( $edit_link ) ? $edit_link : '',
+			'addWorkaroundFor42686' => version_compare( strtok( get_bloginfo( 'version' ), '-' ), '4.9', '==' ),
```

Comment:
We should also make sure the version is under `4.9.2` as soon as we get the fix committed to core.